content/spin/install.md

```diff
@@ -138,12 +138,13 @@ If you have [`cargo`](https://doc.rust-lang.org/cargo/getting-started/installati
 
 <pre class="bash spin-install" id="spin-install-source-cargo"><code>$ git clone https://github.com/fermyon/spin -b v1.0.0
 $ cd spin
+$ rustup target add wasm32-unknown-unknown
 $ rustup target add wasm32-wasi
 $ cargo install --locked --path .
 $ spin --help
 </code></pre>
 
-> Please note: Spin v1.0.0 requires `wasmtime v5.0.0` which requires rustc 1.66.0 or newer. You can update Rust using the following command:
+> Please note: Spin v1.0.0 requires `wasmtime v5.0.0` which requires rustc 1.67.0 or newer. You can update your version of Rust using the following command:
```

Comment:
Is it true that the 1.67 requirement is down to Wasmtime 5 (i.e. the old Rust version was wrong), or is there something else that is now forcing us to Rust 1.67 (i.e. the reason for the new Rust version is different).

It might be better to omit the reason and say only "Spin 1.1 requires Rust 1.67 or above."  Not sure users care which dependency is causing the minimum version requirement.